### PR TITLE
Ear 673 mbs intro panel

### PR DESCRIPTION
**Pull request by jasehumphr**
See https://collaborate2.ons.gov.uk/jira/browse/EAR-673 for the ticket

### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: data/en/mbs_0111.json [hunk 1]
```diff
@@ -44,14 +44,16 @@
...
+
                         "list": ["Data should relate to all sites in England, Scotland and Wales unless otherwise stated.", "You can provide informed estimates if actual figures aren\u2019t available.", "We will treat your data securely and confidentially."]
                     }]
                 }],
                 "preview_content": {
                     "id": "preview",
                     "title": "Information you need",
                     "content": [{
-                        "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                        "description": "<a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/surveys/informationforbusinesses/businesssurveys/monthlybusinesssurveyproductionandservices'>View the survey information and questions before you start the survey (external link)</a>"
```
Comment (MattElcock) [inline data/en/mbs_0111.json]: Both of these, I think, need to be there. You can add another object to the content array with the link in. Just put the link as the first item and the text as the second.

**Review by MattElcock — CHANGES_REQUESTED** [review]

File: data/en/mbs_0204.json [hunk 2]
```diff
@@ -42,14 +42,16 @@
...
+
                         "list": ["Data should relate to all sites in England, Scotland, Wales and Northern Ireland unless otherwise stated.", "You can provide informed estimates if actual figures aren\u2019t available.", "We will treat your data securely and confidentially."]
                     }]
                 }],
                 "preview_content": {
                     "id": "preview",
                     "title": "Information you need",
                     "content": [{
-                        "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate.",
+                        "description": "<a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/surveys/informationforbusinesses/businesssurveys/monthlybusinesssurveyproductionandservices'>View the survey information and questions before you start the survey (external link)</a>"
```
Comment (MattElcock) [inline data/en/mbs_0204.json:54]: You've not added a comma at the end of this line, or in 0203 and 0205 varients of the survey, which (I think) is why Travis is failing.

File: data/en/mbs_0106.json [hunk 3]
```diff
@@ -42,14 +42,16 @@
...
+
                         "list": ["Data should relate to all sites in England, Scotland and Wales unless otherwise stated.", "You can provide informed estimates if actual figures aren\u2019t available.", "We will treat your data securely and confidentially."]
                     }]
                 }],
                 "preview_content": {
                     "id": "preview",
                     "title": "Information you need",
                     "content": [{
-                        "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                        "description": "<a rel='noopener noreferrer' target='_blank' href='https://www.ons.gov.uk/surveys/informationforbusinesses/businesssurveys/monthlybusinesssurveyproductionandservices'>View the survey information and questions before you start the survey (external link)</a>"
```
Comment (jchapman68) [inline data/en/mbs_0106.json:54]: For previous form updates for Covid the (external link) was not part of the link.

**Review by jchapman68 — COMMENTED** [review]
For previous form updates for Covid the (external link) was not part of the link - please check with Jordan

**Review by martyncolmer — APPROVED** [review]

File: data/en/mbs_0106.json (hunk 3 above)
Comment (MattElcock) [reply data/en/mbs_0106.json:54]: The wording is correct, it's on the Jira ticket.

**Review by MattElcock — APPROVED** [review]

**Pull request merged**